ci/test_cpp.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+#!/bin/bash
+
+set -euo pipefail
+
+rapids-logger "Create test conda environment"
+. /opt/conda/etc/profile.d/conda.sh
+
+rapids-dependency-file-generator \
+  --generate conda \
+  --file_key test \
+  --matrix "cuda=${RAPIDS_CUDA_VERSION%.*};arch=$(arch);py=${RAPIDS_PY_VERSION}" > env.yaml
```

Comment:
```suggestion
rapids-dependency-file-generator \
  --generate conda \
  --file_key test \
  --matrix "cuda=${RAPIDS_CUDA_VERSION%.*};arch=$(arch)" > env.yaml
```

We updated this in the `rmm` implementation. Since the `cpp` tests don't depend on any particular Python version, we can remove the value here.